src/ui/pages/SwapForm/fieldsets/ReceiveTokenField/MarketAssetSelect/MarketAssetSelect.tsx

```diff
@@ -42,16 +42,23 @@ export function MarketAssetSelect({
       ),
     [addressPositions]
   );
+
+  const { data: popularAssets, isLoading: popularAssetsAreLoading } = useQuery({
+    queryKey: ['getPopularTokens', chain],
+    queryFn: () => getPopularTokens(chain),
+    suspense: false,
+    retry: false,
+    staleTime: Infinity,
```

Comment:
maybe we can make this longer but still not infinity? i think it can be nice if this data could be updated

Reply:
Fair!